spec/controllers/application_controller_spec.rb

```diff
@@ -122,4 +122,12 @@ def index
       end
     end
   end
+
+  describe '#analytics' do
+    it 'uses the Analytics class by default' do
+      expect(Analytics).to receive(:new).with(controller.current_user, request)
```

Comment:
You don't think that exact args are an implementation detail in the context of this test?   If they are being tested maybe that should be mentioned in the test title?